core/src/main/java/org/apache/brooklyn/core/config/ConfigKeys.java

```diff
@@ -46,6 +47,9 @@
 
     private static final Logger log = LoggerFactory.getLogger(ConfigKeys.class);
     
+    public enum InheritanceContext implements ConfigInheritanceContext
+    { TYPE_DEFINITION, RUNTIME_MANAGEMENT } 
```

Comment:
Personal preference: strange code layout! Can we follow a common coding convention (like that of Oracle, with our agreed variations like line length) when there's not a really good reason to deviate?